frontend/src/pages/EditWorkoutPage.tsx

```diff
@@ -32,14 +32,16 @@ export default function EditWorkoutPage(props: Props) {
             })
                 .then(props.fetchData);
 
-                    navigate("/workouts");
+                    navigate("/");
         }
     }
 
     return (
         <>
+            <div style={{textAlign:"center"}}>
```

Comment:
I'm not sure, but shouldn't we try to do all of the styling in the css-files and not directly in the tsx?